virttest/utils_test/libvirt.py

```diff
@@ -922,6 +922,23 @@ def get_parts_list(session=None):
     return parts
 
 
+def get_disk_by_serial(session, serial_str):
+    """
+    Get disk by serial in VM
+
+    :param session: VM session
+    :param serial_str: ID_SERIAL of disk, string value
+    :return: Disk name if find one with serial_str, else None
+    """
+    parts_list = get_parts_list(session)
+    for disk in parts_list:
+        cmd = ("udevadm info --query=all --name=/dev/{} | grep ID_SERIAL={}"
+               .format(disk, serial_str))
+        if not session.cmd_status(cmd):
```

Comment:
I would suggest to make this method common for host/VM by having session as default argument.
if user gives session use it, else query in host.